### PR TITLE
Feature/nutrition game

### PR DESCRIPTION
**Pull request by kingam280**

### CODE REVIEW

**Review by mateugazi — CHANGES_REQUESTED** [review]
Were you using parcel when you worked on that feature? I don't see that file in "package.json" -> "scripts" -> "start:dev" and in "build" (but that we can add at the end, when we finished the merging). When I add it to the parcel the game couldn't load, because of "Uncaught ReferenceError: regeneratorRuntime is not defined" . We are working on this project with parcel, so I think that should be fixed. I already fixed that in my local repo, but I also don't want to do something on your branch without your allowance, so I'm waiting for your answer

**Comment by kingam280** [comment]
I was working on live server from VS Code but it's fixed and should work now.

**Comment by mateugazi** [comment]
Yes it works now, thanks.

File: src/app/Game.js
Comment (mateugazi) [inline src/app/Game.js:58]: Radek said that we should avoid one letter variables, even when they are obvious

File: src/app/Game.js
Comment (mateugazi) [inline src/app/Game.js:34]: I will copy and paste previous comment: "Radek said that we should avoid one letter variables, even when they are obvious "

**Review by mateugazi — DISMISSED** [review]
To me it's looks cool (I personally don't think that one name variable is a issue here) so you have my approval and btw great job :)

**Review by MaDud — DISMISSED** [review]

File: src/app/Question.js
Comment (kingam280) [reply src/app/Question.js:12]: Okay, I'll add if statement there

**Review by mateugazi — APPROVED** [review]

**Review by mateugazi — DISMISSED** [review]

**Review by MaDud — APPROVED** [review]

**Review by mateugazi — APPROVED** [review]

**Pull request merged**
